4-auth/config.js

```diff
@@ -27,6 +27,8 @@ nconf
     'DATA_BACKEND',
     'GCLOUD_PROJECT',
     'MEMCACHE_URL',
+    'MEMCACHE_USERNAME',
```

Comment:
This isn't one of ours, is it?  we should only be using / showing off our things.